includes/class-wp-job-manager-install.php

```diff
@@ -34,6 +34,12 @@ public static function install() {
 			$is_new_install = true;
 		}
 
+		// On new installs display the usage tracking notice with one week delay and for existing installs display it right away.
+		if ( false === get_option( 'job_manager_display_usage_tracking_once' ) ) {
+			$time_to_show_notice = $is_new_install ? time() + WEEK_IN_SECONDS : time() - 10;
```

Comment:
Not really sure if this simplifies the code but..WDYT?

```suggestion
			$time_to_show_notice = time()  + ( $is_new_install ? WEEK_IN_SECONDS : - 10 );
```

Reply:
Hmm I think that this version is harder to read IMO